File: Clases-Bluej/Veterinario.java
public class Veterinario {
    private String idVeterinario;
    private String nombre;
    private String especialidad;

    public Veterinario(String idVeterinario, String nombre, String especialidad) {
        this.idVeterinario = idVeterinario;
        this.nombre = nombre;
        this.especialidad = especialidad;
    }

    public String getIdVeterinario() { return idVeterinario; }
    public String getNombre() { return nombre; }
    public String getEspecialidad() { return especialidad; }
}

File: Clases_Csharp/Veterinario.cs
namespace SistemaVeterinaria
{
    public class Veterinario
    {
        public string IdVeterinario { get; private set; }
        public string Nombre { get; private set; }
        public string Especialidad { get; private set; }

        public Veterinario(string idVeterinario, string nombre, string especialidad)
        {
            IdVeterinario = idVeterinario;
            Nombre = nombre;
            Especialidad = especialidad;
        }
    }
}